src/utils/parseENSAddress.ts

```diff
@@ -1,7 +1,7 @@
-const ENS_NAME_REGEX = /^(([a-zA-Z0-9]+\.)+)eth(\/.*)?$/
+const ENS_NAME_REGEX = /^(([a-zA-Z0-9]([a-zA-Z0-9-]+[a-zA-Z0-9])\.)+)eth(\/.*)?$/
```

Comment:
does this work with 1-2 character ens names?

Reply:
Nope, let me fix it